packages/app/src/components/AddToLayoutButton/AddToLayoutButton.js

```diff
@@ -47,35 +50,45 @@ export class AddToLayoutButton extends Component {
         this.props.onAddDimension({
             [this.props.dialogId]: axisName,
         });
+        this.props.onUpdate(this.props.ui);
         this.props.closeDialog(null);
     };
 
-    getMenuItems = () =>
-        items.slice(1).map(option => (
+    omitSeriesMenuItem = () =>
```

Comment:
One minor detail - I like function names to be descriptive and lexically / grammatically correct.  So in this case `omit` is a verb, so I think it should be acting on an argument, so say `omitSeriesMenuItem(items)`.  If it instead acts on the `this` object I might make it a method rather than a function and implement `items.withoutSeriesMenuItem()` or something (note that I'm using the preposition `without` instead of the imperative verb conjugation `omit` here)

All this being said, I actually think this is superfluous to implement as a class method.  The result is derived only from a constant, not the state of the AddToLayoutButton class instance, and so should be a constant.  Instead of making this a class method I would just create the constant subset list above, with the other Consts (and maybe consider doing the same for the SERIES and FILTER indexes used below):

```js
const seriesItem = ADD_LAYOUT_OPTIONS[0];
const filterItem = ADD_LAYOUT_OPTIONS[2];
const itemsWithoutSeries = ADD_LAYOUT_OPTIONS.filter(option => option.axisKey !== 'columns');
/*OR, perhaps more descriptive? */ const nonColumnItems = ADD_LAYOUT_OPTIONS.filter(option => option.axisKey !== 'columns');
```

Then just use those const values in the body of the code, since their values will never change.

On another note - I think renaming `ADD_TO_LAYOUT_OPTIONS` to `items` is a little misleading, since it's a constant value while the name `items` implies a variable.  Similarly, I used variable-style names for my consts in this example but might consider something like FILTER_ITEM instead of filterItem (the sliced array is a bit of a gray area since it is the result of a computation on a constant, not a constant itself).

That was a bit long-winded, and this is a very small detail but I think it helps to have code be as readable and understandable as possible for maintainability, so I'm sharing a few of my opinions though I'm sure other people will disagree with some of them!